src/features/student/views/StudentHomeView.vue

```diff
@@ -2,7 +2,9 @@
 import type { ResumeDTO, SkillDTO } from '@/types'
 import { parseDateFR } from '@/common/utils'
 import { StudentOverviewWidget, StudentResumesWidget, StudentSkillsWidget } from '@/features/student/components'
+import { useStudentProfileSummaryQuery } from '@/features/student/queries/use-student-summary.query'
```

Comment:
```suggestion
import { useStudentProfileSummaryQuery } from '@/features/student/queries/'
```

Le projet veut privilégier les barrel files ( https://avenirs-esr.github.io/dev-doc/dev-handbook/ - Clean Code Checklist (cf Uncle Bob) - Point 10) donc mettre un index.ts dans `@/features/student/queries` et exporter les queries d'ici

Reply:
(y)